global/get-modifed-folders/action.yml

```diff
@@ -0,0 +1,196 @@
+name: "📁 Get modified directory and sparse checkout"
+description: "Retrieve all modified directory and perform a sparse checkout."
+
+
+inputs:
+  starting_index:
```

Comment:
@ffppa since you are not passing an actual index (number), i suggest changing the name to a more describing one, like "start_folder"

Reply:
suggesting appliend on [PR-40](https://github.com/pagopa/eng-github-actions-iac-template/pull/40)